SIL.Windows.Forms.Keyboarding/Windows/WinKeyboardAdaptor.cs

```diff
@@ -144,6 +147,20 @@ public void UpdateAvailableKeyboards()
 			}
 		}
 
+		private bool InvalidLanguageName(InputLanguage inputLanguage)
+		{
+			// We have had some input languages get errors when trying to get the layout name, so just want to skip those
+			try
+			{
+				string test = inputLanguage.LayoutName;
+				return false;
+			}
+			catch (Exception)
```

Comment:
Would it be better to catch a specific exception?

I wonder if the exception we get here is the one we caught previously which got refactored/removed in bfedd4ae811261b585e13bc7823a62781a8c11cb.

Reply:
There is no documented exception for InputLanguage.LayoutName, so that's why I just caught anything.

We are still accessing the property that caused the Paratext problem report, so even with the other changes that I did recently, this error is still possible.